File: csci294/2020Spring/inclass/wk04_2020-02-07_fri/GuessingGame.cpp
/* Guessing Game */

#include <iostream>
#include <cstdlib> // srand(), rand, RAND_MAX
#include <ctime> // time()
using namespace std;

int main() {
	srand(time(0));
	int number{ rand() % 100 + 1 }; // 1 - 100
	cout << "Hint: the number is " << number << endl;
	int guess{ 0 };
	bool done{ false };
	cout << "I'm thinking of a nuber from 1 to 100." << endl;
	cout << "Can you guess the number?" << endl;

	cout << "Make your first guess: ";
	while (!done) {
		cin >> guess;
		if (guess < number) {
			cout << "Too low. Guess again: ";
		}
		else if (guess > number) {
			cout << "Too high. Guess again:";
		}
		else {
			cout << "Good Job." << endl;
			done = true;
		}
	}

	return EXIT_SUCCESS;
	


}

File: csci195/2020fall/inclass/wk12_2020-11-06_fri/guessing-game.py
# guessing-game.py

import random

def main():
    n = random.randint(1, 10)
    print(f'CHEAT ==> The number is {n}')
    
    guess = int(input('Guess a number from 1 to 10: '))
    
    finished = guess == n
    
    while not finished:
        if guess < n:
            prompt = 'Too low. Guess again: '
        else:
            prompt = 'Too high. Guess again: '
            
        guess = int(input(prompt))
        finished = guess == n
    
    print('Congrats! You got it')
    
main()